HeterogeneousCore/CUDAUtilities/interface/HistoContainer.h

```diff
@@ -0,0 +1,161 @@
+#ifndef HeterogeneousCoreCUDAUtilitiesHistoContainer_h
+#define HeterogeneousCoreCUDAUtilitiesHistoContainer_h
+
+
+#include<cassert>
+#include<cstdint>
+#include<algorithm>
+#ifndef __NVCC__
+#include<atomic>
+#endif
+#ifdef __NVCC__
+#include <cuda_runtime.h>
```

Comment:
I would #include this also when `__NVCC__` / `__CUDACC__` are not defined; in those cases it will define `__host__`, `__device__`, `__global__` etc. to nothing.

Reply:
ok